modules/appnexusBidAdapter.js

```diff
@@ -331,14 +331,16 @@ export const spec = {
       payload.referrer_detection = refererinfo;
```

Comment:
is it worth to also condition out the variable 
`const VIDEO_MAPPING`
`const VIDEO_TARGETING`
`const VIDEO_RTB_TARGETING`
in line 45, 46, 55

Reply:
Since `let` and `const` are block-scoped you'd have to do something like this to avoid ESLint errors:

```js
let VIDEO_TARGETING,
  VIDEO_RTB_TARGETING,
  VIDEO_MAPPING;

if (FEATURES.VIDEO) {
  VIDEO_TARGETING = [ /* ... */ ];
  VIDEO_RTB_TARGETING = [ /* ... */ ];
  VIDEO_MAPPING = { /* ... */ }
}
```

which actually results in a bundle that's bigger by 4 bytes.